src/js/apps/globals/app-frame_app.js

```diff
@@ -20,9 +20,10 @@ export default App.extend({
     this.currentWorkspace = Radio.request('bootstrap', 'currentWorkspace');
     this.getRegion('content').empty();
 
+    new NavApp({ region: this.getRegion('nav') });
+
```

Comment:
This fixes the issue with the incorrect workspace being shown in the top nav button (this story: [sc-34124](https://app.shortcut.com/roundingwell/story/34124/bug-showing-incorrect-current-workspace-in-top-nav-menu-button)). Which only occurs after the user switches workspaces.

But I'm wary of unintended consequences of this...